<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="OverviewTab">
         <attribute name="title">
          <string>Overview</string>
         </attribute>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>160</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </widget>
        </widget>
        <widget class="QWidget" name="EmployeeTab">
         <attribute name="title">
          <string>Employee Tab</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="3">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QListView" name="employeeListView">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <property name="resizeMode">
               <enum>QListView::Adjust</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QFrame" name="formFrame">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="firstNameLabel">
               <property name="text">
                <string>First Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="firstNameLineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lastNameLabel">
               <property name="text">
                <string>Last Name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lastNameLineEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="genderLabel">
               <property name="text">
                <string>Gender</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="genderComboBox">
               <item>
                <property name="text">
                 <string>Male</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Female</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="jobPositionLabel">
               <property name="text">
                <string>Job Position</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="jobPositionLineEdit"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="streetAddressLabel">
               <property name="text">
                <string>Street Address</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="streetAddressLineEdit"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="cityLabel">
               <property name="text">
                <string>City</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="cityLineEdit"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="stateLabel">
               <property name="text">
                <string>State</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="stateLineEdit"/>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="zipcodeLabel">
               <property name="text">
                <string>Zipcode</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="zipcodeLineEdit"/>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="hourlyWageLabel">
               <property name="text">
                <string>Hourly Wage</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLineEdit" name="hourlyWageLineEdit"/>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="numberOfHoursWorkedLabel">
               <property name="text">
                <string>Number Of Hours Worked</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLineEdit" name="numberOfHoursWorkedLineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="editButton">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 3</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAuthor"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
